### DISCUSSION TITLE
Show field acreage in course generation screen

### DISCUSSION THREAD

**Discussion by crcarmichael** [Courseplay/courseplay#7153] [Ideas]
After lots of field merging and expansion, it's difficult to determine acreage of any given field.  While this info isn't required, it's nice to know.  It seem to me that this information is inherently calculable during field scanning and/or course generation and it would be nice to display this on the course generation screen.

**Comment by Tensuko** [comment]
Have a look here #6463

**Reply by crcarmichael** [reply]
That's not what a mean... I mean just a calculated area in acres/ha.  So, in the course generation screen, click on a field (or select with the arrows) and it displays the acreage for that field: 9.87ac, for example.  This could be shown as an overlay on the generation map itself or in the generation settings panel.
Like I said, it's not *necessary* information, but I'm always curious how many acres I'm working.

**Comment by Tensuko** [comment]
That is already implemented.
Not the most beautiful color, but trust me, it was hard to find something that is half way readable on most PDA backgrounds...
![fsScreen_2021_04_28_17_49_04](https://user-images.githubusercontent.com/27344744/116434045-5a3e2300-a84a-11eb-9188-6a10628be5e5.png)

**Reply by dete96** [reply]
Awesome! That's why I read every single post here. I use CP for a long time now, but never saw this. Great!
Thank you for your hard work.

**Reply by crcarmichael** [reply]
That's great!
